lib/pender_data.rb

```diff
@@ -11,7 +11,8 @@ def validate_pender_result(force = false, retry_on_error = false)
       result = { type: 'error', data: { code: -1 } }.with_indifferent_access
       pender_key = get_pender_key
       begin
-        result = PenderClient::Request.get_medias(CheckConfig.get('pender_url_private'), params, pender_key)
+        timeout = CheckConfig.get('short_request_timeout') if self.is_a?(Media) && self.original_claim.present?
```

Comment:
Suggestion: Add a default value and make clear that you expect an integer value:

```ruby
CheckConfig.get('short_request_timeout', 25, :integer)
```